### DISCUSSION TITLE
incorrect export result

### DISCUSSION THREAD

**Discussion by detkliff** [dbeaver/dbeaver#29944] [Q&A] [answered]
### Description

incorrect export query result 

### DBeaver Version

community edition 24.0.4

### Operating System

windows 11 64bit

### Database and driver

Postgresql 14.11

### Steps to reproduce

1. Create table
CREATE TABLE public.nsi_candidate_group11 (
	candidate_group_id int4 GENERATED BY DEFAULT AS IDENTITY( INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 START 316 CACHE 1 NO CYCLE) NOT null,
	candidate_group_code text NOT NULL
);
2. Insert data
![image](https://github.com/dbeaver/dbeaver/assets/61137576/f87b4a06-9fa9-482e-8703-a468848eb621)

4. Select data
![image](https://github.com/dbeaver/dbeaver/assets/61137576/034b691b-8036-4dc7-ab22-c5dd0cfd7d89)

5.export result data
![image](https://github.com/dbeaver/dbeaver/assets/61137576/05f877e2-3838-4ea2-a682-bd8fee48052a)
![image](https://github.com/dbeaver/dbeaver/assets/61137576/af1e3717-8955-42c6-aede-25be20a1cabb)
![image](https://github.com/dbeaver/dbeaver/assets/61137576/933f628f-61aa-4c06-accf-482224d9a0e3)
![image](https://github.com/dbeaver/dbeaver/assets/61137576/48fc2a2d-6f63-41f2-8b86-ab956f172596)

![image](https://github.com/dbeaver/dbeaver/assets/61137576/90e27dbb-7b7f-4a38-b039-c5000c02f65f)

6. insert data (ctrl+v)

![image](https://github.com/dbeaver/dbeaver/assets/61137576/c3ec6fcd-9b9d-4a58-836f-e144684e6665)

In result one column only, but selected two columns candidate_group_id and candidate_group_code

**Comment by detkliff** [comment]
export to file similar effect

**Comment by E1izabeth** [comment, marked as answer]
If you want to include ids to the export result, you need to check `Include generated columns` option
![image](https://github.com/dbeaver/dbeaver/assets/28875055/4de8c088-c01f-4d4a-9e82-99173e3d00e6)
